function/runtime.go

```diff
@@ -51,6 +53,9 @@ func (env *ExecutionEnvironment) Execute(data interface{}) error {
 	if err := env.addSearch(vm); err != nil {
 		return err
 	}
+	if err := env.addSendMail(vm); err != nil {
```

Comment:
You preferred a separate function to adding it to the `addHelpers` existing function?

Reply:
yes